docsets/PyTorch/docset.json

```diff
@@ -12,7 +12,15 @@
                 "Machine Learning",
                 "Neural Networks"],
     "specific_versions": [
-            {
+        {
+            "version": "0.3.0",
+            "archive": "versions/0.3.0/PyTorch.tgz",
+            "author": {
```

Comment:
Authors for specific versions aren't currently supported. This doesn't do anything (it's ignored by Dash).